README.md

```diff
@@ -17,21 +17,16 @@ Install [`frint-cli`](https://frint.js.org/docs/packages/frint-cli/):
 $ npm install -g frint-cli
 ```
 
-Create a new empty directory:
-
-```
-$ mkdir my-directory && cd my-directory
-```
-
 Initialize an example app:
 
 ```
-$ frint init --example kitchensink
+$ frint new my-directory --example=kitchensink
```

Comment:
do we need the `=` in the command?

we use `yargs` in `frint-cli`, which would already parse the values for us:

```
$ frint new my-directory --example kitchensink
```

Reply:
`yargs` can parse values from both structures. Anyway, I have 0 preference towards either of them, so I'm good with changing it.

Reply:
thanks! current documentation is not using `=` anywhere. so would be great if we can keep it that way everywhere.